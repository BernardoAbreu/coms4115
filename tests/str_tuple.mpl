fun string_of_tuple(tup,separator) {
	str = tup$(0);

	i = 1;
	while(i < len(tup)) {
		str = str + separator + tup$(i) ;
		i = i+1;
	}

	return str;
}



tuple Person {name, age:integer, address}

b = Person;
b$name = "Bernie";
b$age = 21;
b$address = "Bauxita Street";

print(string_of_tuple(b, ", "));