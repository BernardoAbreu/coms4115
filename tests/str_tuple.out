Bernie, 21, Bauxita Street
